tests/test_config_entries.py

```diff
@@ -554,9 +554,9 @@ class TestFlow(config_entries.ConfigFlow, domain="test"):
 
             VERSION = 5
 
-            async def async_step_discovery(self, user_input=None):
+            async def async_step_discovery(self, discovery_info):
                 """Test discovery step."""
-                if user_input is not None:
+                if discovery_info is not None:
```

Comment:
Can we remove this check?

Reply:
Hehe no, we can't, since basically that test cannot happen...
I will rewrite that thing.

Reply:
Done.